cli/admin/src/test/java/io/pravega/cli/admin/AbstractTlsAdminCommandTest.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.cli.admin;
+
+import io.pravega.client.ClientConfig;
+import io.pravega.client.stream.impl.DefaultCredentials;
+import io.pravega.local.LocalPravegaEmulator;
+import io.pravega.test.common.SecurityConfigDefaults;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.Timeout;
+
+import java.net.URI;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+public abstract class AbstractTlsAdminCommandTest {
+
+    // Security related flags and instantiate local pravega server.
+    protected static final AtomicReference<AdminCommandState> STATE = new AtomicReference<>();
```

Comment:
Why is this static? 

Do not use static fields that you set during your execution. This will prevent us from eventually running tests in parallel.

Reply:
I've made this field non-static as suggested.